wandb/sdk/launch/_project_spec.py

```diff
@@ -118,7 +118,8 @@ def __init__(
         if override_entrypoint:
             _logger.info("Adding override entry point")
             self.override_entrypoint = EntryPoint(
-                " ".join(override_entrypoint[0]), override_entrypoint
```

Comment:
Not sure what this was intended to do originally, but I don't think it was used anywhere... This would always give me something like `"p y t h o n"`. I changed it to track the entrypoint file instead.

Reply:
Old comment on L115 seems to imply `entry_point` was once expected to be a list of lists of strings in the name of supporting multiple entry points, but it doesn't look like this function would handle that correctly anyway.